src/privateai_client/components/request_objects.py

```diff
@@ -459,40 +437,56 @@ class ProcessedMarkerText(BaseRequestObject):
         "UNIQUE_NUMBERED_ENTITY_TYPE",
         "UNIQUE_HASHED_ENTITY_TYPE",
     ]
+    default_marker_language = "auto"
```

Comment:
The default is "en" under the docs. I think we should not turn this on by default like the web demo.

Reply:
ah shoot, you're right.. I don't know why I saw default as auto for some reason :/